pycfmodel/model/resources/properties/security_group_egress_prop.py

```diff
@@ -18,6 +18,9 @@
 from .property import Property
 
 
+import ipaddress
+
+
 class SecurityGroupEgressProp(Property):
     CidrIp: Optional[ResolvableStr] = None
     CidrIpv6: Optional[ResolvableStr] = None
```

Comment:
We might be able to change this to be of type IPv4 and IPv6 address, although unsure if this would have other side effects: https://pydantic-docs.helpmanual.io/usage/types/

Would it look like `CidrIp: Optional[ResolvableStrOrIpv4] = None` where `ResolvableStrOrIpv4` = `Resolvable[Union[str, ipaddress.Ipv4Address]]`